components/admin_console/system_user_detail/system_user_detail.jsx

```diff
@@ -225,33 +309,60 @@ export default class SystemUserDetail extends React.Component {
                 </div>
                 <div className='admin-console__wrapper'>
                     <div className='admin-console__content'>
-                        <AdminPanel
-                            titleId='(First Name) (Last Name) • (Nickname)'
-                            titleDefault={firstName + ' ' + lastName + ' | ' + nickname}
-                            subtitleId='(Position)'
-                            subtitleDefault={user.position ? user.position : ''}
+                        <AdminUserCard
+                            userFirstLastNickname={firstLastNickname}
```

Comment:
Why not pass the user into `AdminUserCard` directly instead of passing in 3 different arguments related to it? 0/5 either way, but it seems a bit simpler the other way

Reply:
I was wondering that as well as this evolved. Should be cleaner.